### ISSUE TITLE
chaincode package golang

### ISSUE THREAD

**Issue by channingduan** [hyperledger/fabric-admin-sdk#94]

**Comment by bestbeforetoday** [comment]
I don't think we should be spending time implementing support for the legacy (Fabric v1.x) chaincode packaging.

The Fabric v2.x chaincode packaging is a [well-defined tar/gzip file structure](github.com/hyperledger/fabric/gossip/discovery). Tools and libraries already exist for creating tar/gzip files so I don't see a lot of value in providing explicit support for creating them in the admin SDK either.

**Comment by davidkhala** [comment]
> I don't think we should be spending time implementing support for the legacy (Fabric v1.x) chaincode packaging.
> 
> The Fabric v2.x chaincode packaging is a [well-defined tar/gzip file structure](github.com/hyperledger/fabric/gossip/discovery). Tools and libraries already exist for creating tar/gzip files so I don't see a lot of value in providing explicit support for creating them in the admin SDK either.

Maybe we can consider 1 option if we really want to provide something here: A validator (func or binary) that make sure the provided tar/gzip is under well-defined file structure.

**Comment by bestbeforetoday** [comment]
Closing this as we are not targeting the deprecated chaincode deployment mechanism, and there seems to be no end-user demand for this.
